apps/full-site-editing/full-site-editing-plugin/starter-page-templates/class-starter-page-templates.php

```diff
@@ -203,7 +203,7 @@ public function fetch_vertical_data() {
 
 		// Load fresh data if we don't have any or vertical_id doesn't match.
 		if ( false === $vertical_templates || ( defined( 'WP_DEBUG' ) && WP_DEBUG ) ) {
-			$vertical_id = get_option( 'site_vertical', 'default' );
+			$vertical_id = get_option( 'site_vertical' ) ?: 'default';
```

Comment:
The linter doesn't like the short ternary here.

Reply:
I noticed that, too, but there seem to be plenty pf other instances in ET. 
Could we make the linter a little less strict on that?

/cc @noahtallen

Reply:
Probably fine! We just extend the WordPress rules for the most part. So long as it's compatible with wpcom simple & atomic PHP versions, it should be fine. should be able to override the rule here: https://github.com/Automattic/wp-calypso/blob/master/apps/phpcs.xml

Reply:
though it would probably also be good to check that this is accepted by the wpcom linter too. I'm honestly not sure. 

> there seem to be plenty pf other instances in ET.

I think most php in the plugin unfortunately side-stepped the lint rules at some point, so a lint check on the whole plugin fails in a lot of places. 🙃 The rules should be the source of truth for linting going forward rather than the code (and we can obviously change the rules to accomplish that too!)

Reply:
funny enough, someone mentioned this on the support page for the plugin (which we don't really check) https://wordpress.org/support/topic/incorrect-minimum-php-version/

Reply:
> I noticed that, too, but there seem to be plenty pf other instances in ET.

I can't seem to find any 😮